jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java

```diff
@@ -0,0 +1,68 @@
+package io.github.com;
+
+import com.epam.jdi.light.common.CheckTypes;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.pageobjects.annotations.JSite;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import io.github.com.pages.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'io.github.com.pages'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

Reply:
this import used for private HomePage homePage;
all imports are used